include/toolchain/common.h

```diff
@@ -98,7 +98,8 @@
 /* Indicate that an array will be used for stack space. */
 
 #if !defined(_ASMLANGUAGE)
-  #define __stack __aligned(STACK_ALIGN)
+  /* don't use this anymore, use K_DECLARE_STACK instead. Remove for 1.11 */
```

Comment:
yes we do, similar to the linux kernel:

#if ZEPHYR_VERSION_CODE <= ZEPHYR_VERSION(1,8,0)
foo
#else 
bar
#endif